src/components/graphDiagram/widgets/ViaNodeWidget.tsx

```diff
@@ -115,52 +106,52 @@ export const ViaNodeWidget: React.FC<ViaNodeProps> = ({model, engine}) => {
                         }, index: number) => (
-                            <React.Fragment key={index}>
+                              <React.Fragment key={index}>
                                 <Stack
-                                    padding=".5rem"
-                                    spacing={1}
-                                    direction="row"
-                                    alignItems="center"
+                                  padding=".5rem"
+                                  spacing={1}
+                                  direction="row"
+                                  alignItems="center"
+                                  borderTop={index !== 0 ? '1px solid #9BA2B0' : 0}
                                 >
-                                    {item.type === NodeTypes.Origin &&
-                                      <OriginIcon fill="#088AB2" width={"1rem"} height={"1rem"}/>}
-                                    {item.type === NodeTypes.Via &&
-                                      <ViaIcon fill="#088AB2" width={"1rem"} height={"1rem"}/>}
-                                    <Typography
-                                        sx={{
-                                            color: "#667085",
-                                            fontSize: "0.875rem",
-                                            fontWeight: 500,
-                                            lineHeight: "1.25rem",
-                                        }}
-                                    >
-                                        {item.name}
-                                    </Typography>
+                                  {item.type === NodeTypes.Origin &&
+                                      <OriginIcon fill="#6C707A" width={"1rem"} height={"1rem"}/>}
+                                  {item.type === NodeTypes.Via &&
+                                      <ViaIcon fill="#6C707A" width={"1rem"} height={"1rem"}/>}
+                                  <Typography
+                                    sx={{
+                                      color: "#6C707A",
+                                      fontSize: "0.875rem",
+                                      fontWeight: 400,
+                                      lineHeight: "1.25rem",
+                                    }}
+                                  >
+                                    {item.name}
+                                  </Typography>
                                 </Stack>
                                 {index < (model.getOptions().from?.length ?? 0) - 1 && <Divider/>}
-                            </React.Fragment>
+                              </React.Fragment>
                         ))}
                     </Box>
-
+                  <Box
+                    style={{
+                      width: "1rem",
+                      height: "0.0625rem",
+                      backgroundColor: " #6C707A",
+                      transform: "rotate(90deg)",
```

Comment:
@Salam-Dalloul Can replace width and height , instead of rotate.
width: 0.0625rem,
height: 1rem